include/substrait-mlir/Dialect/Substrait/IR/SubstraitOps.td

```diff
@@ -362,8 +362,8 @@ def Substrait_LiteralOp : Substrait_ExpressionOp<"literal", [
     ```
   }];
   // TODO(ingomueller): extend to other types.
-  let arguments = (ins Substrait_AtomicAttribute:$value);
-  let results = (outs Substrait_AtomicType:$result);
+  let arguments = (ins AnyAttrOf<[Substrait_AtomicAttribute,Substrait_ContainerAttribute]>:$value);
```

Comment:
Maybe it would make sense to add a `def` to `SubstraitAttrs.td` called something like `Substrait_LiteralAttribute`?